tests/merch.test.ts

```diff
@@ -0,0 +1,63 @@
+import { ForbiddenError } from 'routing-controllers';
+import { UserAccessType } from '../types';
+import { DatabaseConnection, MerchFactory, UserFactory, PortalState } from './data';
+import { ControllerFactory } from './controllers';
+
+beforeAll(async () => {
+  await DatabaseConnection.connect();
+});
+
+beforeEach(async () => {
+  await DatabaseConnection.clear();
+});
+
+afterAll(async () => {
+  await DatabaseConnection.clear();
+  await DatabaseConnection.close();
+});
+
+describe('collection editing', () => {
+  test('Set collection to archive', async () => {
+    const conn = await DatabaseConnection.get();
```

Comment:
This test case description doesn't really describe what we are asserting. We're essentially making that sure that hidden items cannot be ordered from, so let's make that clear in the name.

Reply:
Also, I think this test is asserting two different things, and can probably be split into two test cases related to archiving collections to keep each of the cases as simple and clear as possible. One can be for checking if getting an archived collection does or does not throw an error for admins and non-admins, and the other can be for checking if hidden items can be ordered from.